test/spec/geo/ui/time_slider.spec.js

```diff
@@ -63,6 +63,20 @@ describe('cdb.geo.ui.TimeSlider', function() {
       });
     });
 
+    describe("given a step that spans more than 48 hours", function() {
+      beforeEach(function() {
+        var start = 1423699205000;
+        var end = 1424649534000;
+        view.torqueLayer.getTimeBounds = function(){return {start: start, end: end}};
+        view.torqueLayer.options = {steps: 5};
+        formatter = view.formatterForRange.bind(view)(start, end);
```

Comment:
why doing the bind here? since you're calling the `formatterFoRange` on the view object the `this` context will be the view